stravalib/client.py

```diff
@@ -263,7 +262,7 @@ def get_athlete(self):
         :rtype: :class:`stravalib.model.Athlete`
         """
         raw = self.protocol.get("/athlete")
-        return model.Athlete.deserialize(raw, bind_client=self)
+        return model.Athlete.parse_obj(raw)
```

Comment:
This is an example of how to deserialize the raw response dict into the new pydantic-based classes